### ISSUE TITLE
Add GitLab Cookie Cutter usage in Project Creation

### ISSUE THREAD

**Issue by gmr** [aweber/imbi#12]
After #11 is complete, add the automation for applying a Cookie Cutter against the GitLab project.

**Comment by dave-shawley** [comment]
@gmr closed by https://github.com/AWeber-Imbi/imbi/pull/59 ?

**Comment by gmr** [comment]
Closing as no longer relevant: The cookiecutter automation approach from the v1 project creation pipeline was replaced in Imbi 2.0 by the `imbi-automations` workflow system. Project creation automations in v2 follow a different model.
